src/main/java/core/basesyntax/model/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Square extends Color {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + getArea() + " sq. units, side: "
+                + side + " units, color: " + getColor());
```

Comment:
The `draw` method is conceptually different from the `getArea` method. According to the checklist, you should not put all behavior into a single interface if the methods are conceptually different. Consider separating these responsibilities into different interfaces.